browser/flagr-ui/src/components/Flag.vue

```diff
@@ -896,6 +934,60 @@ export default {
         this.$message.success("variant updated");
       }, handleErr.bind(this));
     },
+    createTag() {
+      Axios.post(
+        `${API_URL}/flags/${this.flagId}/tags`,
+        this.newTag
+      ).then((response) => {
+        let tag = response.data;
+        this.newTag = clone(DEFAULT_TAG);
+        if (!this.flag.tags.map((tag) => tag.value).includes(tag.value)) {
+          this.flag.tags.push(tag);
+          this.$message.success("new tag created");
+        }
+        this.tagInputVisible = false;
+        this.loadAllTags();
+      }, handleErr.bind(this));
+    },
+    cancelCreateTag() {
+      this.newTag = clone(DEFAULT_TAG);
+      this.tagInputVisible = false;
+    },
+    queryTags(queryString, cb) {
+      let results = this.allTags.filter((tag) => tag.value.toLowerCase().includes(queryString.toLowerCase()));
+      cb(results);
+    },
+    loadAllTags() {
+      Axios.get(
+        `${API_URL}/tags`
```

Comment:
I noticed that even when we delete the tag from across flags, it still shows up in the autocomplete.

Reply:
This is actually intended for the Flags delete, because it simply removes the reference to the Tag from the Flag and not the actual Tag itself because it could be referenced by another Flag. Let us know if there's another way you'd like to handle this.

Reply:
@zhouzhuojie, I was talking to @bradphilips and a couple of our other Flagr users, and we wondered if this would be a preferred way to handle this anyway. It would keep us from creating and deleting the same tag over and over if it goes in and out of use.

Brad also mentioned an option to search for any flags with the removed tag and fully delete the tag if there aren't any. I can see an argument for doing that to prevent unused tags from sticking around.

Personally, I like leaving them the way it's currently set up, but I don't think we'd argue if you want to fully delete the tags.